Extensions/dnSpy.Debugger/dnSpy.Debugger.DotNet.CorDebug/Code/DbgDotNetNativeLocationImpl.cs

```diff
@@ -69,5 +69,9 @@ sealed class DbgDotNetNativeCodeLocationImpl : DbgDotNetNativeCodeLocation {
 			NativeAddress.Offset == other.NativeAddress.Offset;
 
 		public override int GetHashCode() => Module.GetHashCode() ^ (int)Token ^ (int)Offset ^ (int)ILOffsetMapping ^ NativeAddress.Address.GetHashCode() ^ NativeAddress.Offset.GetHashCode() ^ CorCode.HashCode;
+
+		public int CompareTo(DbgDotNetNativeCodeLocationImpl other) {
+			return this.CompareTo((DbgDotNetNativeCodeLocation)other);
```

Comment:
I don't use `this.` unless required by the compiler.

The cast is not needed.

Simple one-line methods like this one should use `=>` (the `=>` should be on the same line as the method name)